fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/api/LogicalResourceIdentKey.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * (C) Copyright IBM Corp. 2020, 2021
```

Comment:
was it moved from somewhere else?  if not, just 2022
```suggestion
 * (C) Copyright IBM Corp. 2022
```

Reply:
fixed